src/plupload.js

```diff
@@ -1473,6 +1473,11 @@ plupload.Uploader = function(settings) {
 						file.name = fileName;
 					}
 					
+// 					// The below should replace all chars with potential issues from here:
```

Comment:
This should run for all uploaded files based on the comment on 1470.

Reply:
This is also a preemptive fix to cover potential future issues if AWS suddenly decides to enforce other bad characters.

Reply:
Would love to catch these before they hit pluploader tbh. But nice to have a final check here.

Reply:
Fair, though I feel this is a bit more dry. We won't have to add this code to every frontend that can do uploads.